spec/spec_helper.rb

```diff
@@ -16,4 +16,8 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  # disable monkey patching
+  # see: https://relishapp.com/rspec/rspec-core/v/3-8/docs/configuration/zero-monkey-patching-mode
```

Comment:
URL doesn't work for me.

```suggestion
  # https://rspec.info/features/3-12/rspec-core/configuration/zero-monkey-patching-mode/
```